src/modules/users/resolvers/signup.js

```diff
@@ -16,11 +16,7 @@ UserTC.addResolver({
     const user = await User.signup(args.record)
     return {
       recordId: user._id,
-      record: {
-        _id: user._id,
-        fullName: user.fullName,
-        email: user.email
-      }
+      record: user
```

Comment:
Returning `user` directly will return all fields, including the password - which we don't want. Please return the necessary fields only

Reply:
password will not be returned

Reply:
You should only return the fields needed - not all of them. The `signup` static method returns the entire `User` object, which you just returned here - so it will return all fields

Reply:
okay done